docs/contributing/coding-style.md

```diff
@@ -776,6 +778,34 @@ tests := map[string]func(t *testing.T) test {
     }
     ```
 
+#### Input agreement
+
+1. test case name
+
+    Test case name should be readable and meaningful for understanding each test easily.
+    If you create a new test, the test name should be named based on the below-naming templates.
+
+    - Success:
+        - Start with `success` or `{verb} success` or `success {verb}`
+        - End with the condition `when {condition}` or `with {condition}
+        - e.g.: `Set success when value is default value`
```

Comment:
Which is correct, `Set` or`Sets`? 🤔  
You used `Returns` in the `Returns: ` section, so I think  `Sets` is correct 🤔 

And Wich is correct, `set` or `Set`? 🤔

Reply:
I think `set` is more correct.

Reply:
```suggestion
        - e.g.: `set success when the value is default value`
```